File: dayTwo/Java/After Break/Test/TestQuestionTwo.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class TestQuestionTwo{

    @Test
    void testThatWhenIEnterThreeValuesIGetAGradeThatIsTheAverage(){
        int firstScore = 90;
        int secondScore = 90;
        int thirdScore = 90;
        
        char expectedAnswer = QuestionTwo.grade(firstScore,secondScore,thirdScore);
        char actualAnswer = 'A';
        
        assertEquals(expectedAnswer, actualAnswer);    
    }
}

File: dayTwo/python/After Break/test_question_two.py
import unittest

from question_two import grade

class TestGrade(unittest.TestCase):
    
    def test_thatWhen_iEnterThreeScores_IGetTheGrade_OfTheAverage_ofTheScores(self):
        expectedAnswer = grade(90,90,90)
        
        actualAnswer = 'A'
        
        self.assertEqual(expectedAnswer,actualAnswer)
        
